deployments/helm/nvidia-device-plugin/values.yaml

```diff
@@ -35,6 +35,7 @@ deviceIDStrategy: null
 nvidiaDriverRoot: null
 gdsEnabled: null
 mofedEnabled: null
+imexNodesConfigFile: null
```

Comment:
I can see us wanting to add additional IMEX options. Does something like:
```
imex:
  nodesConfigFile: null
```
make sense to allow for extension?

Reply:
if we open this option, I would like to also have a `enabled` var.

Reply:
@klueska any thoughts?

Reply:
`imex` is complex enough for us to have additional options associated with it. We were, for example, discussing having the injection of the default channel off-by-default in the plugin. Making this a config object (?) instead of a single property makes this cleaner to implement.